### ISSUE TITLE
Prefixes missing from json-ld context

### ISSUE THREAD

**Issue by kshefchek** [biolink/biolink-model#128]
- HGNC

**Comment by cmungall** [comment]
@hsolbrig this should be derived from id_prefixes in the model

we have this:

```
  gene:
    is_a: gene or gene product
    aliases: ['locus']
    mappings:
      - "SO:0000704"
      - "SIO:010035"
      - "WD:Q7187"
      - "UMLSSG:GENE"
    id_prefixes:
      - NCBIGene
      - ENSEMBL
      - HGNC
      - MGI
      - ZFIN
      - dictyBase
      - WB
      - SGD
      - PomBase
```

any idea why this one is skipped in the jsonld context?

**Comment by hsolbrig** [comment]
Fixed in pull request #130.  (Note that request incorrectly names issue #119...)
